syspurpose/man/syspurpose.8

```diff
@@ -48,6 +48,9 @@ Add values to the add-ons property
 \fBsyspurpose remove-addons \fIVALUE\fP...\fP
 Remove values from the add-ons property
 .TP
+\fBsyspurpose unset-addons \fIVALUE\fP...\fP
+Clear set add-ons property
```

Comment:
I would write "Clear the system addons property". addons property is a set, but it looks confusing.